doc/tutorial.rst

```diff
@@ -182,37 +186,36 @@ the `SimpleVideoBlock.href` attribute. The view is defined as a method on the XB
             return hostname, response['html']
 
 The HTML we obtained from the provider is interpolated into a HTML template
```

Comment:
Nit, since we're here: 'an HTML template', rather than 'a HTML' template. H is pronounced 'a-che', so the leading vowel sound requires 'an' instead of 'a'.